css/wfrp4e.css

```diff
@@ -826,7 +826,7 @@
 
   #sidebar.collapsed {
     width: 50px !important;
-    height: 364px !important;
+    height: 400px !important;
```

Comment:
Wouldn't this apply even if the module is not active?

Reply:
without the height extension, we lose the last icon :D
<img src="https://puu.sh/IHjjY/d87a91cddb.png">

Reply:
Right, but with the height extension, what does it look like without the module?

Reply:
Hoo sorry ! on my first message, you have the 2 first screens without the module but already with the height extension actived

Reply:
Ah sorry I misunderstood, thanks.

Reply:
No problem I don't speak a lot too :D